modules/thermal_hydraulics/doc/content/modules/thermal_hydraulics/modeling_guide/recuperated_brayton_cycle/recuperated_brayton_cycle.md

```diff
@@ -0,0 +1,222 @@
+
+
+
+### PID Control
+
+Shown below is the `ControlLogic` block containing the PID block utilizing [PIDControl.md]. Here, the $K_p$, $K_i$, and $K_d$ variables are the proportional, integral, and derivative gains respectively. As with any PID control, gains must be determined for a specific system and are not universal. In this specific PID, the output signal is used to govern the torque supplied by the motor to the system shaft based upon the current shaft speed.
+
+!listing thermal_hydraulics/test/tests/problems/brayton_cycle/recuperated_brayton_cycle.i
+       block=ControlLogic/initial_motor_PID
+       link=False
+
+The PID attempts to reach a desired shaft speed, in RPM, which is determined by the `set_point` block using [GetFunctionValueControl.md]. The rated speed of the turbine is set at the beginning of the input file as a global parameter `speed_rated_rpm`. To prevent over-speeding of the turbine and compressor, the set point of the PID is adjusted to be lower than the rated turbine speed.
+
+!listing thermal_hydraulics/test/tests/problems/brayton_cycle/recuperated_brayton_cycle.i
+       block=ControlLogic/set_point
+       link=False
+
+A [ParsedFunctionControl.md] is applied to determine whether to send the PID output to the motor or shutdown the motor because the turbine is supplying enough torque.
+
+!listing thermal_hydraulics/test/tests/problems/brayton_cycle/recuperated_brayton_cycle.i
+       block=ControlLogic/logic
+       link=False
+
+If `logic` determines to send the PID signal to the motor, then [SetComponentRealValueControl.md] `motor_PID` is used to apply the signal directly to the `motor` component torque.
+
+!listing thermal_hydraulics/test/tests/problems/brayton_cycle/recuperated_brayton_cycle.i
+       block=ControlLogic/motor_PID
+       link=False
+
+## Motor Shutdown
+
+Should the turbine provide more torque than the motor, the logic activates a [MooseParsedFunction.md], `motor_torque_fn_shutdown`, to ramp down the motor allowing for a graceful transition of the system from the motor to the turbine.
+
+!listing thermal_hydraulics/test/tests/problems/brayton_cycle/recuperated_brayton_cycle.i
+       block=Functions/motor_torque_fn_shutdown
+       link=False
+
+The shutdown function utilizes `AuxVariables` values `PID_trip_status` and `time_trip`. These values are dependent upon a number of factors including the motor PID, startup rate of the heat source, heat source total power, etc. The user has no way of knowing these values while writing the input file, so they must be determined during input file execution. The following steps outline one method to determine, store, and apply initially unknown variables within an input file.
+
+
+## Motor Shutdown: AuxVariables
```

Comment:
For the following sub sections, add one level and remove "Motor Shutdown" from the title -> 
'### AuxVariables'